doc/source/data/loading-data.rst

```diff
@@ -790,35 +714,62 @@ Call :func:`~ray.data.read_sql` to read data from a database that provides a
                 "SELECT year, COUNT(*) FROM movie GROUP BY year", create_connection
             )
 
+Reading NoSQL databases
+~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
just call this Reading from MongoDB, since we don't currently support other NoSQL databases?